Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -82,6 +82,7 @@ class TextureCacheBase
     bool is_custom_tex;
     bool may_have_overlapping_textures = true;
     bool tmem_only = false;  // indicates that this texture only exists in the tmem cache
+    bool has_manual_mips = false; // indicates that the mips in this texture are arbitrary content, aren't just downscaled
```

Comment:
Is there a reason you went with `manual_mip` for all of your function/variable names?  I feel like something else (arbitrary_mips or non_uniform_mips?) would be more explanatory.

Reply:
Honestly I just changed my mind constantly about the name. Inconsistent/arbitrary mips sounds ok too! Not sold on "non_uniform" though :P